wp-content/themes/pub/wporg-learn-2020/css/vendor/_overrides.scss

```diff
@@ -14,12 +14,17 @@ body:not(.trac):not(.home-page) #main {
 
 h1, h2, h3, h4 { color: #555 !important; }
 
-/* ___ Homepage ___ */
+section {
+	padding-top: 0;
 
-#page.home {
-	background-color: #fff;
+	+ section,
+	+ hr + section {
```

Comment:
Add the large spacing back if this is a layout broken into sections, including where an hr is used between.